src/app/@ansyn/ol/plugins/annotations/annotations-context-menu/components/annotation-context-menu/annotation-context-menu.component.html

```diff
@@ -1,19 +1,21 @@
-<ng-container *ngFor="let featureId of selection">
-	<div class="context-menu-wrapper"
-		 *ngIf="annotations?.idToEntity?.has(featureId)"
-		 [ngStyle]="calcBoundingRect(featureId)">
+<div *ngIf="show">
```

Comment:
1  Check with Tal whether to close to the contxt menu on export (I think yes)
2 If you still close the context menu, use ng-container instead of div